hbase-common/src/main/java/org/apache/hadoop/hbase/io/crypto/Encryption.java

```diff
@@ -229,7 +229,7 @@ public static boolean failOnHashAlgorithmMismatch(Configuration conf) {
    * Return the SHA-256 digest of the concatenation of the supplied arguments.
    */
   public static byte[] hash256(String... args) {
-    return hashWithAlg("SHA-256", args);
+    return hashWithAlg("SHA-256", Bytes.toByteArrays(args));
```

Comment:
Same as in L218

Reply:
the method `hashWithAlg` needs a vararg array of byte arrays to operate on. This call here is coercing the vararg array of Strings into an array of byte arrays. AFAIK it's needed.